src/issue-store.js

```diff
@@ -199,7 +199,30 @@ const issueStore = new class IssueStore extends EventEmitter {
     }
     let fetchPromise;
     if (Client.canCacheLots()) {
-      fetchPromise = Client.getOcto().repos(repoOwner, repoName).issues.fetchAll(opts);
+      if (Client.useGraphQL()) {
+        const [issues, pullRequests] = await Promise.all([
+          Client.getGraphQLClient()
+            .repo(repoOwner, repoName)
+            .issues({sort: 'UPDATED_AT', earliestDate: lastSeenAt})
+            .fetchAll({per_page: 100}),
+          Client.getGraphQLClient()
+            .repo(repoOwner, repoName)
+            .pullRequests({sort: 'UPDATED_AT', earliestDate: lastSeenAt})
+            .fetchAll({per_page: 30})
+        ]);
+        let result = [];
+        if (issues) {
+          result = result.concat(issues);
+        }
+        if (pullRequests) {
+          result = result.concat(pullRequests);
+        }
+        fetchPromise = new Promise(resolve => resolve(
```

Comment:
There's `Promise.resolve()`

But returning whatever directly also wraps it  into promise because it's inside an async function.

Reply:
`Promise.resolve()` works. 👍 
`fetchPromise = result.map((item) => item.issue);` This doesn't work though, not sure if I was doing wrong.